lib/objects/objectsInMemFileDB.js

```diff
@@ -53,6 +53,9 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
 
         this.objectsDir = path.join(this.dataDir, 'files');
 
+        // cached meta objects for file operations
+        this.metaCache = {};
```

Comment:
Maybe name it better "existingMetaObjects" or such? ... Cache is relative unspecific for the use case (now that I see it in code ...)

Reply:
And we do not cache the object but just the "is meta type" status, so no cache ...

Reply:
Sounds good